modules/control-utility/src/test/java/org/apache/ignite/util/GridCommandHandlerTest.java

```diff
@@ -606,13 +607,37 @@ private void setState(Ignite ignite, ClusterState state, String strState, String
      */
     @Test
     public void testBaselineCollect() throws Exception {
-        Ignite ignite = startGrids(1);
+        Ignite ignite = startGrid(
+            optimize(getConfiguration(getTestIgniteInstanceName(0))).setLocalHost("0.0.0.0"));
+
+        Field addresses = ignite.cluster().node().getClass().getDeclaredField("addrs");
+        addresses.setAccessible(true);
+        addresses.set(ignite.cluster().node(), Arrays.asList("127.0.0.1", "0:0:0:0:0:0:0:1", "10.19.112.175", "188.166.164.247"));
```

Comment:
Is it theoretically possible to create node with both empty addresses and hostnames? If true lets write test for such case.